matrix_client/api.py

```diff
@@ -254,6 +254,76 @@ def send_message_event(self, room_id, event_type, content, txn_id=None,
             params["ts"] = timestamp
         return self._send("PUT", path, content, query_params=params)
 
+    def send_call_invite(self, room_id, call_id, sdp, version, lifetime,
+                         msgtype="m.call.invite", timestamp=None):
```

Comment:
`msgtype` kwarg is never used. Probably just shouldn't be included in method signature at all.